app/questionnaire/value_source_resolver.py

```diff
@@ -100,12 +100,41 @@ def _resolve_list_item_id_for_value_source(
             else None
         )
 
+    def resolve_dynamic_answers(
+        self,
+        answer_id: str,
+    ) -> Generator[AnswerValueTypes, None, None]:
+        # Type ignore: the answer block will exist at this stage
+        question = self.schema.get_block_for_answer_id(answer_id).get("question", {})  # type: ignore
+
+        if not (dynamic_answers := question.get("dynamic_answers")):
+            return
+
+        values = dynamic_answers["values"]
+        if values["source"] == "list":
+            for list_item_id in self.list_store[values["identifier"]].items:
```

Comment:
Does it not work the same without `.items`?

Reply:
it does indeed, good point, removed .items